collective/trustedimports/suds.py

```diff
@@ -0,0 +1,33 @@
+from collective.trustedimports.util import whitelist_module
+import os
+from AccessControl import allow_class, ModuleSecurityInfo, ClassSecurityInfo, Unauthorized
+from AccessControl.class_init import InitializeClass
+from Products.PythonScripts.Utility import allow_module
+from suds.client import Client as _Client
+from suds import WebFault
+from suds import MethodNotFound
+
+
+class SafeClient(_Client):
+    """ SafeClient
+    """
+
+    def __init__(self, url, **kwargs):
+
+        blacklist = os.getenv('SAFEIMPORTS_URL_BLACKLIST ', '')
```

Comment:
needs to support wildcards.
probably should be in the utils for other libraries that access urls to use

Needs to only do the blacklist when being called via a restrictedpython same as https://github.com/collective/collective.trustedimports/blob/master/collective/trustedimports/safezipfile.py#L20
Maybe move that check function into utils

Reply:
@nngu6036 I've written you a helper function in utils to make this much easier. You just need to check the rest of the functions to make sure they are safe and write the tests

Reply:
@djay , I add the test